package.json

```diff
@@ -38,35 +38,40 @@
     "access": "public"
   },
   "dependencies": {
-    "expo": "^49.0.0",
+    "expo": "^50.0.7",
     "phoenix": "^1.7.6",
     "promise-fs": "^2.1.1"
   },
   "devDependencies": {
+    "@babel/core": "7.23.9",
+    "@babel/eslint-parser": "7.23.10",
+    "@babel/plugin-proposal-private-property-in-object": "7.21.11",
+    "@babel/preset-env": "7.23.9",
+    "@react-native/babel-preset": "0.73.21",
```

Comment:
do we need all of this? shouldn't those packages be added by other packages?

Reply:
removed